definitions/checks/disk_speed_minimal.rb

```diff
@@ -0,0 +1,9 @@
+class Checks::DiskSpeedMinimal < ForemanMaintain::Check
+  for_feature :disk_io
+  description 'Check for recommended disk speed of pulp, mongodb, pgsql dir.'
```

Comment:
On newer versions (checks with sat62), the hdparm command can not be present (needs to be pulled via hdparm rpm). I suggest putting 

```ruby
confine do
  execute?("which hdparm")
end
```

to make it valid only when the tool is in place (and we will ensure the hdparm is there via rpm depednencies.